### PR TITLE
Run flake8 across this codebase

### PR DESCRIPTION
**Pull request by alexmuller**
flake8 spotted that there was a duplicate entry in the dictionary so it's probably worth running as part of the tests.

I don't know tox at all so I may have done this in the wrong place.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,3 +1,6 @@
+[flake8]
+max-line-length=100
```
Comment (gforcada) [inline setup.cfg]: Would you mind changing that to 80? I usually work with two or three editors one next to each other and 80 is the perfect size for that.

File: tox.ini [hunk 2]
```diff
@@ -9,6 +9,8 @@ deps=
     pytest
     pytest-cov
     coveralls
+    flake8
 commands=
+    flake8
```
Comment (gforcada) [inline tox.ini]: That does not work, you should instead put ``flake8 *.py``

**Review by gforcada — CHANGES_REQUESTED** [review]
Thanks for spending the time cleaning up my mess!! 🎉 

There's a couple of small changes I would like to make on your changes, if you wish I can do them myself if you want. Just let me know.

Again, thanks for caring so much to send a couple of pull requests already!!

**Comment by alexmuller** [comment]
No problem at all! Updated this pull request, sorry for the delay :)

**Comment by gforcada** [comment]
@alexmuller thanks!  I will release it shortly, ping me back if I did not do it.

**Comment by gforcada** [comment]
@alexmuller released 1.3 let me know if something is broken 🙂

**Pull request merged**